deploy/50_cloud-ingress-operator.Deployment.yaml

```diff
@@ -67,4 +71,24 @@ spec:
           - mountPath: /etc/pki/ca-trust/extracted/pem
             name: trusted-ca-bundle
             readOnly: true
-            
\ No newline at end of file
+        - name: cio-admission
+          # Replace this with the built image name
+          image: quay.io/twilliams725/cloud-ingress-operator:latest
+          command:
+          - cioadmission
+          - "--secure-port=5000"
+          - "--tls-cert-file=/var/webhook-certs/tls.crt"
+          - "--tls-private-key-file=/var/webhook-certs/tls.key"
+          imagePullPolicy: Always
+          ports:
+          - containerPort: 5000
+            protocol: TCP
+          volumeMounts:
```

Comment:
Probably need to mount the trusted-ca-bundle configmap here to fix the problem in the description.